specification/machinelearning/data-plane/Azure.AI.ChatProtocol/preview/2023-10-01-preview/examples/create.json

```diff
@@ -0,0 +1,36 @@
+{
+  "operationId": "GenericChatClient_Create",
+  "title": "Creates a chat completion for the provided prompt using streaming.",
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "api-version": "2023-10-01-preview",
+    "body": {
+      "messages": [
+        {
+          "content": "Tell me a joke",
+          "role": "user"
+        }
+      ],
+      "stream": false
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "choices": [
+          {
+            "index": 0,
+            "message": {
+              "content": "Why did the tomato turn red?\nBecause it saw the salad dressing!",
+              "role": "assistant",
+              "session_state": null
```

Comment:
We do not want a `null` session_state.

Reply:
I don't have something that would look real to put there, but I'll add something :)

Reply:
I think we want to have something json like here. Or possibly a base64 encoded string.

Reply:
already fixed it, forgot to tsp compile :/